src/renderer/actions/connections.js

```diff
@@ -61,15 +61,16 @@ export function connect (id, databaseName, reconnecting = false, sshPassphrase)
       dispatch({ type: CONNECTION_REQUEST, server, database, reconnecting, isServerConnection: !databaseName });
 
       if (!serverSession) {
-        if (server.ssh.privateKeyWithPassphrase && typeof sshPassphrase === 'undefined') {
-          dispatch({ type: CONNECTION_REQUIRE_SSH_PASSWORD });
-          return;
+        if (server.ssh) {
```

Comment:
ok i've left the  if ( server.ssh && ...) from a previous test, 
but do you prefer a 
```js
if ( ssh ) {
if (a) {}
if (b) {}
}
```
or 
```js
if ( ssh && a ) {}
if ( ssh && b) {}
```

personally i think i prefer the first one ....